astropy/coordinates/representation.py

```diff
@@ -2027,9 +2093,10 @@ def represent_as(self, other_class, differential_class=None):
         if inspect.isclass(other_class) and not differential_class:
             if issubclass(other_class, SphericalRepresentation):
                 return other_class(lon=self.phi, lat=90 * u.deg - self.theta,
-                                   distance=self.r)
+                                   distance=self.r, copy=False)
```

Comment:
definitely better, but those are done in #11470 too, so seems good to just leave it for there...

Reply:
rolling back. 🔙

Reply:
✅